src/components/tabs/_tabs.scss

```diff
@@ -90,57 +91,78 @@
           padding-bottom: govuk-spacing(2);
         }
 
-        @include govuk-if-ie8 {
-          // IE8 doesn't support `box-shadow` so add an actual border that is
-          // modified to indicate state.
-          border-top: 4px solid transparent;
+        // Since borders are joined with a  45 degree angle bevel 'mitre joint',
+        // we can't set the top border color.
+        //
+        // +-------------+
+        // | \=========/ |
+        // |   +-----+   |
+        // |   | Tab |   |
+        // +---+     +---+
+        //
+        // To get around this we float a border above the tab
+        // so that it'll be 90 degree angle 'butt joint'.
+        //
+        // +---+-----+---+
+        // |   |=====|   |
+        // |   +-----+   |
+        // |   | Tab |   |
+        // +---+     +---+
+        //
+        &:after {
+          content: "";
+          position: absolute;
+          top: -1px;
+          right: 0;
+          left: 0;
+          border-width: 0;
+          border-style: solid;
+          border-color: transparent;
         }
 
         &:hover {
-          box-shadow: inset 0 4px 0 0 govuk-colour("light-blue");
+          // Remove outline from unenhanced focus state
+          outline: none;
+          // Remove styles from unenhanced focus state
+          box-shadow: none;
 
-          @include govuk-if-ie8 {
+          &:after {
+            border-top-width: 4px;
             border-top-color: govuk-colour("light-blue");
           }
         }
 
         &--selected {
-          $top-border-width: 1px;
-          $top-border-adjustment: 1px;
-
-          @include govuk-if-ie8 {
-            // IE8 already has a top border so no adjustment is needed
-            $top-border-width: 4px;
-            $top-border-adjustment: 0;
-          }
-
-          margin-top: - govuk-spacing(1);
+          margin-top: - (govuk-spacing(1) - 1px);
           margin-bottom: -1px;
 
-          // 1px is compensation for border (otherwise we get a 1px shift)
```

Comment:
Non-blocking: this comment still feels useful (maybe minus the reference to 1px) for why we're reducting pixels from the padding.

Reply:
This comment no longer applies since the border is always around.